ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -2320,6 +2320,58 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: CIS - Ensure the Hide IP Address in Safari is Enabled (Optional)
+  platforms: macOS
+  platform: darwin
+  description: |
+    Public (Routable) IP addresses can be used to track people to their current location, including
+    home and business addresses. While a valid IP addess is necessary to load the site the valid
+    address does not need to be provided to known trackers and should be hidden.
+  resolution: |
+    Perform the following steps to set Safari whether or not to hide IP addresses from trackers:
+    1. Open Safari
+    2. Select Safari from the menu bar
+    3. Select Settings
+    4. Select Privacy
+    5. Set Hide IP address from trackers to Enabled
```

Comment:
Wouldn't it be better to give the terminal method since it's automatic?

Ask your .... to run this ... 

$ /usr/bin/sudo -u <username> /usr/bin/defaults write
/Users/<username>/Library/Containers/com.apple.Safari/Data/Library/Preference
s/com.apple.Safari WBSPrivacyProxyAvailabilityTraffic -int <130272/130276>